Feature#105.2 create the edit sptcialty page

andrii-sharun opened the pull request:
dev
CLICKUP

https://app.clickup.com/t/fwzn9d


## Code reviewers
@v-ilchenko 
@Vasyl3611 
@Ormared 

## Summary of issue
Created a page for editing information about the specialty

## Summary of change
Created a page for editing information about the specialty

## Testing approach
Test UI components in a user-centric way.

## CHECK LIST
- [ ]  СI passed
- [+]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

andrii-sharun committed:
create edit specialty page and custom input component

andrii-sharun committed:
Edit styles, refactor code

v-ilchenko commented:
Let input-borders be rounded) as in Figma, as in other pages style

Whole styles some differs from Figma's "EditSpecialty" style, but it's ok

Maybe add some vertical padding to teaxArea field

Should it be pointer cursor on labels hover in main info block?

andrii-sharun committed:
small style changes, clear code

andrii-sharun committed:
clear code

andrii-sharun commented:
> Let input-borders be rounded) as in Figma, as in other pages style
> 
> Whole styles some differs from Figma's "EditSpecialty" style, but it's ok
> 
> Maybe add some vertical padding to teaxArea field
> 
> Should it be pointer cursor on labels hover in main info block?

andrii-sharun commented:
> Let input-borders be rounded) as in Figma, as in other pages style
> 
> Whole styles some differs from Figma's "EditSpecialty" style, but it's ok
> 
> Maybe add some vertical padding to teaxArea field
> 
> Should it be pointer cursor on labels hover in main info block?

Fixed! Made rounding for input fields, added vertical padding for textarea and 'cursor: pointer' rule for labels style was  deleted.
